src/bicep/modules/SentinelSolution.bicep

```diff
@@ -0,0 +1,20 @@
+param workspaceName  string
+param workspaceId string
+param location string = resourceGroup().location
+param tags object = {}
+
+resource azurerm_log_analytics_solution 'Microsoft.OperationsManagement/solutions@2015-11-01-preview'={
+  name: 'SecurityInsights(${workspaceName})'
+  location: location
+  tags:tags
+  
+  properties: {
+    workspaceResourceId: workspaceId
+  }
+  plan: {
+    name: 'SecurityInsights(${workspaceName})'
+    publisher: 'Microsoft'
+    product: 'OMSGallery/SecurityInsights'
+    promotionCode: ''
+  }
+}
```

Comment:
The modules at `/modules` are camelCased named, lets we rename this to `sentinelSolution`

Reply:
Updated the module name to comply with camelCase format